### PR TITLE
AMB-16 Project Dao

### PR DESCRIPTION
**Pull request by CharlotteMKainos**
created project dao and changed data type values in project class

### CODE REVIEW

**Review by dana-kainos — APPROVED** [review]

**Review by dana-kainos — CHANGES_REQUESTED** [review]
what table is it from

**Review by dana-kainos — APPROVED** [review]

File: src/main/java/org/soniakbew/daos/ProjectDao.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package org.soniakbew.daos;
+
+import org.soniakbew.models.Project;
+import org.soniakbew.models.ProjectProperties;
+
```
Comment (lukemKainos) [inline src/main/java/org/soniakbew/daos/ProjectDao.java]: space?

File: src/main/java/org/soniakbew/daos/ProjectDao.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                resultSet.getInt("technologyId"),
+                resultSet.getInt("techLeadId"),
+                resultSet.getInt("clientId"),
+                resultSet.getInt("salesEmployeeId"),
+                new ProjectProperties(
+                resultSet.getDate("startDate"),
+                resultSet.getDate("finishDate"),
+                resultSet.getFloat("commissionRate"),
+                resultSet.getDouble("value")
+                ));
```
Comment (lukemKainos) [inline src/main/java/org/soniakbew/daos/ProjectDao.java]: can you seperate space this closing bracket?

File: src/main/java/org/soniakbew/daos/ProjectDao.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    }
+
+    public List<Project> getAllProjects() throws SQLException {
+        List<Project> projects = new ArrayList<>();
+
+        try (Connection connection = DatabaseConnector.getConnection()) {
```
Comment (lukemKainos) [inline src/main/java/org/soniakbew/daos/ProjectDao.java:36]: this connection should be closed after use

**Review by lukemKainos — CHANGES_REQUESTED** [review]

**Review by lukemKainos — APPROVED** [review]

**Pull request merged**